src/test/java/org/cactoos/iterable/RangeOfTest.java

```diff
@@ -55,12 +55,13 @@ public final void testIntegerRange() {
     public final void testIntegerFibonacciRange() {
         MatcherAssert.assertThat(
             "Can't generate a range of fibonacci integers",
-            new CollectionOf<>(
+            new ListOf<>(
                 new RangeOf<>(
                     1,
                     100,
                     new Func<Integer, Integer>() {
                         private int last;
+
```

Comment:
@fanifieiev is it really necessary to add this empty line?

Reply:
@iakunin fixed